core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/utils/UrlHelper.kt

```diff
@@ -0,0 +1,25 @@
+package com.pierfrancescosoffritti.androidyoutubeplayer.core.player.utils
+
+/**
+ * Extracts the video ID from a YouTube URL.
+ *
+ * @param url The YouTube URL.
+ * @return The video ID, or the original URL if the video ID cannot be found.
+ */
+fun getVideoIdFromUrl(url: String): String? {
+    val startIndex = url.indexOf("?v=")
```

Comment:
I think the rest of the code is formatted using two spaces instead of tabs, please be consistent. Same for all the other files.

Reply:
I used IDE formatter, hope it be fine now, if you have a rules of formation or linting please share to apply

Reply:
I still see a 4 spaces indentation, it should be 2 spaces. I think this change is simple enough, does not require specific formatting rules.